api-reference/50 Common/Object Structures/positionConfig/boundary.md

```diff
@@ -4,7 +4,9 @@ type: String | Node | jQuery | Window
 ---
 ---
 ##### shortDescription
-The element within which the widget is positioned.
+A boundary element within which the overlay element should be positioned.
```

Comment:
Can you clarify what you're trying to say here? What is a "boundary element" vs an "overlay element"?

Reply:
Let's say:
A boundary element in which an overlay element must be positioned.